hack/charts/2.0.0/images.json

```diff
@@ -0,0 +1,27 @@
+{
+   "pachd": {
+      "repository": "pachyderm/pachd",
+      "tag": "latest",
```

Comment:
Just checking: should this become `2.0.0-beta.8` since that's what we're getting approval for?